CTOR-1083 : Connectors 24.12 Release - Release Note (#3871)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md

- [**AWS Transit Gateway**](../procedures/cloud-aws-transitgateway.md) - Added default value `awscli` in host template custom mode.
- [**AWS VPN**](../procedures/cloud-aws-vpn.md) - Added default value `awscli` in host template custom mode.
- [**Fortinet Fortimanager SNMP**](../procedures/network-fortinet-fortimanager-snmp.md) - Added `device-policy-package-status thresholds` to the command `Net-Fortinet-Fortimanager-SNMP-Device-Status` (from [TheWatch Idea)](https://thewatch.centreon.com/ideas/fortimanager-monitoring-connector-device-status-integrate-policy-package-in-monitoring-connector-4022).
- [**HP Procurve SNMP**](../procedures/network-switchs-hp-procurve-snmp.md) - Added new mode stack from community contribution [PR 5082](https://github.com/centreon/centreon-plugins/pull/5082).
- [**Keysight NVOS Rest API**](../procedures/network-keysight-nvos-restapi.md) - Enhanced ports mode and added license mode.
- [**Nokia TiMos SNMP**](../procedures/network-nokia-timos-snmp.md) - Added new mode sas-alarm from community contribution [PR 5083](https://github.com/centreon/centreon-plugins/pull/5083).

> - [**Windows Centreon Monitoring Agent**](../procedures/operatingsystems-windows-centreon-monitoring-agent.md) - Added Natives checks for CPU, Cpu-detailed, Memory, Storage, Swap and Uptime (This native checks will be available with the Collect release coming in the next few days).

Reviewer: J'attends des nouvelles de l'équipe Collect sur quand a lieu la release de l'Agent pour avoir les natives checks

> - [**Windows Centreon Monitoring Agent**](../procedures/operatingsystems-windows-centreon-monitoring-agent.md) - Added Natives checks for CPU, Cpu-detailed, Memory, Storage, Swap and Uptime (This native checks will be available with the Collect release coming in the next few days).

Reviewer: ```suggestion
- [**Windows Centreon Monitoring Agent**](../procedures/operatingsystems-windows-centreon-monitoring-agent.md) - Added Natives checks for CPU, Cpu-detailed, Memory, Storage, Swap and Uptime (This native checks will be available with the Collect 24.10.3 release coming soon).
```

</TabItem>
<TabItem value="Breaking changes" label="Breaking changes">

> - [**Linux SSH**](../procedures/operatingsystems-linux-ssh.md) - Fixed an issue with process mode doesn't returning the process in some cases by adding limit column size to avoid filter issues.

Reviewer: @omercier 
Je veux bien ton avis sur cette formulation vu que tu es celui qui le plus suivi ce ticket ^^'

</TabItem>
<TabItem value="Fix" label="Fix">

> - [**Azure Elastic Pool**](../procedures/cloud-azure-database-elasticpool.md) - Fixed an issue with storage mode where thresholds wasn't taken into account.

Reviewer 2: ```suggestion
- [**Azure Elastic Pool**](../procedures/cloud-azure-database-elasticpool.md) - Fixed an issue with **storage** mode where thresholds weren't taken into account.
```

> - [**Commvault Commserve Rest API**](../procedures/applications-commvault-commserve-restapi.md) - Fixed storagepools mode to avoid division by 0, from community feedback [PR 5141](https://github.com/centreon/centreon-plugins/pull/5141).

Reviewer 2: ```suggestion
- [**Commvault Commserve Rest API**](../procedures/applications-commvault-commserve-restapi.md) - Fixed **storagepools** mode to avoid division by 0, from community feedback [PR 5141](https://github.com/centreon/centreon-plugins/pull/5141).
```
